src/View.Uno/Extensions/ControlAutoFocus/ControlExtensions.AutoFocus.cs

```diff
@@ -1,209 +0,0 @@
-#if WINDOWS_UWP || __ANDROID__ || __IOS__ || __WASM__
-using System;
-using System.Collections.Generic;
-using System.Text;
-using System.Threading.Tasks;
-using Windows.UI.Xaml.Controls;
-using Windows.UI.Core;
-using Windows.UI.Xaml;
-
-namespace Nventive.View.Extensions
-{
-	public partial class ControlExtensions
-	{
-		/// <summary>
-		/// Get value of IsAutofocusingOnce
-		/// </summary>
-		/// <param name="obj">Control</param>
-		/// <returns>Value of IsAutofocusingOnce</returns>
-		public static bool GetIsAutofocusingOnce(DependencyObject obj)
-		{
-			return (bool)obj.GetValue(IsAutofocusingOnceProperty);
-		}
-
-		/// <summary>
-		/// Set value of IsAutofocusingOnce
-		/// </summary>
-		/// <param name="obj">Control</param>
-		/// <param name="value">New value of IsAutofocusingOnce</param>
-		public static void SetIsAutofocusingOnce(DependencyObject obj, bool value)
-		{
-			obj.SetValue(IsAutofocusingOnceProperty, value);
-		}
-
-		/// <summary>
-		/// Property for IsAutofocusingOnce, wich only triggers the first time the control is displayed 
-		/// </summary>
-		public static readonly DependencyProperty IsAutofocusingOnceProperty =
-			DependencyProperty.RegisterAttached("IsAutofocusingOnce", typeof(bool), typeof(ControlExtensions), new PropertyMetadata(false, IsAutofocusingOnceChanged));
```

Comment:
Is there an equivalent for this? `IsAutofocusingOnce` is used in projects.

Reply:
for the 

> `PathControl `is widely used. Are we sure we want to remove it? The alternative `PathIcon `is not a `FrameworkElement`, so I don't understand how they're equivalent.

PathIcon inherits from iconElement and it inherits from FrameworkElement :/
reference: https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.pathicon?view=winrt-22000

but maybe you mean that could be too complicated to replace all the reference we have from PathControl with this Uno contol?

Reply:
> Is there an equivalent for this? `IsAutofocusingOnce` is used in projects.

nope... that prop is not implemented on WCT

Reply:
Right, I confused with `IconElement `and `IconSource`.

Reply:
But `PathControl` was stylable and `IconElement` can't because it's not a `Control`.

Reply:
removed from PR `PathControl` and `ControlAutoFocusExtension`